lib/chip-types/collection.js

```diff
@@ -149,7 +152,7 @@ Collection.prototype.validate_field_values = function(context, assume_delete_val
 		}
 		const user_errors_amount = Object.keys(user_errors).length;
 		if (user_errors_amount > 0){
-			throw new Errors.ValidationError("There are problems with some of the provided values.", user_errors);
+			throw new Boom.badRequest("There are problems with some of the provided values.", user_errors);
```

Comment:
I think that `forbidden` is more appropriate here. `badRequest` is for requests malformed on a syntax-level (like json issues or smth). If we're executing the code, then the request has been parsed successfully, so there's no need for a `400 Bad Request`.  See [reference](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.4)